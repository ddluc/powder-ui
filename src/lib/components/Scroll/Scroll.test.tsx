import React from 'react';
import { render, cleanup } from '@testing-library/react';
import { Scroll } from './index';

describe('Scale', () => {
  afterEach(cleanup);

  it('should render with overflow-x', () => {
    const result = render(<Scroll direction="x" />);
    const component = result.container.firstChild;
    expect(component).toMatchSnapshot();
  });

  it('should render with overflow-y', () => {
    const result = render(<Scroll direction="y" />);
    const component = result.container.firstChild;
    expect(component).toMatchSnapshot();
  });
});
