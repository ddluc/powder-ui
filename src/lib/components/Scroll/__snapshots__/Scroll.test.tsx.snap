// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Scale should render with overflow-x 1`] = `
.c0 {
  display: block;
  height: auto;
  width: auto;
  position: relative;
}

.c1 {
  overflow-x: scroll;
}

<div
  class="c0 c1"
  direction="x"
/>
`;

exports[`Scale should render with overflow-y 1`] = `
.c0 {
  display: block;
  height: auto;
  width: auto;
  position: relative;
}

.c1 {
  overflow-y: scroll;
}

<div
  class="c0 c1"
  direction="y"
/>
`;
